news/2016-10-10-tip2-devmode.asciidoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: news
+title: Weld Tip &num;2 - Development Mode
+author: Martin Kouba
+priority: 1
+change_frequency: weekly
+excerpt: Meet the Weld Probe - a development tool which allows to inspect CDI components at runtime.
```

Comment:
Ok, I'm not sure here because of all those exceptions (rivers etc.). But it sounds good without `the`.